### PR TITLE
WW-4793 only add JBossFileManager when supported

### PR DESCRIPTION
**Pull request by sdutry**

### CODE REVIEW

File: core/src/main/java/org/apache/struts2/util/fs/JBossFileManager.java [hunk 1]
```diff
@@ -210,4 +187,33 @@ private void addIfAbsent(List<URL> urls, URL fileUrl) {
         }
     }
 
+    public static class JBossFileManagerSupportUtil {
```
Comment (aleksandr-m) [inline core/src/main/java/org/apache/struts2/util/fs/JBossFileManager.java]: Why inner class? Why not just static methods in `JBossFileManager`?

Comment (sdutry) [inline core/src/main/java/org/apache/struts2/util/fs/JBossFileManager.java]: @aleksandr-m 
Good point, let me change that.

Comment (sdutry) [inline core/src/main/java/org/apache/struts2/util/fs/JBossFileManager.java]: @aleksandr-m 
Changed to static methods.
(don't know why i didn't just do that from the start)

**Comment by sdutry** [comment]
Given the input on the issue, this pull request should not be merged.

**Pull request closed without merging**